app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt

```diff
@@ -44,20 +44,18 @@ import io.github.esentsov.PackagePrivate
 enum class NavigationItem(
     @PackagePrivate
     internal val primaryRoute: String,
-    private val canonicalRoute: String,
+    private val canonicalRoute: String = primaryRoute,
     val arguments: List<NamedNavArgument> = emptyList(),
```

Comment:
I think it can be removed because it isn't used anywhere now as we declare arguments directly in `getRouteWithArgs` and we don't have any non-string arguments - what do you think?